hermes/views.py

```diff
@@ -78,19 +87,57 @@ def validate_brightness_unit(self, data):
             if row['brightness_unit'] not in brightness_units:
                 raise ValidationError(f'Unrecognized brightness unit. Accepted brightness units are {brightness_units}')
 
-
-class CandidateMessageSubSchema(Schema):
-    authors = fields.String()
-    main_data = fields.Nested(CandidateDataSchema)
-
-
-class CandidateMessageSchema(Schema):
+class MessageSchema(Schema):
     title = fields.String(required=True)
     topic = fields.String(required=True)
     event_id = fields.String()
     message_text = fields.String(required=True)
-    author = fields.String(required=True)
-    data = fields.Nested(CandidateMessageSubSchema)
+    submitter = fields.String(required=True)
+    authors = fields.String()
+    @property
+    @abstractmethod
+    def data(self):
+        pass
+
+
+class CandidateMessageSchema(MessageSchema):
+    data = fields.Nested(CandidateDataSchema)
+
+
+class PhotometryDataSchema(Schema):
+    target_name = fields.String()
+    ra = fields.String(required=True)
+    dec = fields.String(required=True)
```

Comment:
I feel like we should require either a valid target name, OR an RA/DEC. 
We don't currently include RA/DEC on the frontend message table.
Also, RA/DEC is mostly meaningless for moving object photometry.

Reply:
Valid target name is the rub. The MPC has this figured out. No one else does. The GW community had like 7 names for AT 2017gfo. This was a specific requests from Andy. 

On the moving object front, that is a good point. I have no idea how to handle both cases given the statement about names above.

Reply:
Can it be that if we don't get an RA/DEC we perform a Catalog Search like we do for the observing portal with maybe a SN DB thrown in, and if that turns up nothing, send back as invalid?

Reply:
Maybe? I think it's a bigger can of worms than it's worth.

Reply:
Since this is for a reduced datum, then it corresponds to an image that was taken at a specific moment in time, at an apparent ra/dec even if it was a moving target, right? So I feel like RA/DEC could be universal since its the apparent ra/dec where and when the data point was taken.

Reply:
Depends on how fast the target is moving and how close it is. The RA/DEC of a moving target changes during the exposure, and is different from different sites.

The point is more that it's not useful, and generally not recorded in photometry tables. This makes including it as a required field problematic.

Reply:
Maybe we can take a step back here.  Each science community has it's own conventions for identifying targets, and the users know what those conventions are.  E.g. if I'm working in a project for asteroids, I would use a naming convention that I know I can throw at the MPC and configure my TOM accordingly.  If I'm working in supernovae, I'm expecting a sidereal object with a name that should be recognized by TNS.  But the users are likely going to have to indicate which service they expect to interact with and follow the relevant naming convention.  So some name identifier should be required, and RA/Dec or orbital elements may be provided when relevant.

Reply:
We should include the reference epoch for the coordinate system in use though, e.g. J2000.0.  This does make a difference e.g. for high proper motion objects.  The parameter should be optional however.